modules/social_features/social_follow_content/social_follow_content.routing.yml

```diff
@@ -0,0 +1,7 @@
+social_follow_content.overview:
+  path: '/following'
+  defaults:
+    _controller: '\Drupal\social_follow_content\Controller\Overview::handle'
+    _title: 'Content and posts I follow'
```

Comment:
I think just keeping this "Content I follow" as it is now works fine because `posts` is included in the term "content". This also saves us translating a new string.